README.md

```diff
@@ -1 +1,169 @@
-# Hello_Microverse-
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello_Microverse-] <a name="about-project"></a>
+
+**[Hello_Microverse-]** is a first project in Microverse.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+I'm using mainly html, css for building this project.
+
+ <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Home_Page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming Soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install text editor of your choice e.g VS Code, Atom, etc.
+- Version Control System (Git Preferred).
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/AlselwiAisha/Hello_Microverse-.git
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+- Open cloned repository in your text editor and start editing as required.
+
+### Usage
+
+To run the project, execute the following command:
+
+- Open index.html page of cloned repository in your local browser.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+
+```sh
+  npx eslint .
+```
+
+### Deployment
+
+You can deploy this project using:
+- github pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **AishaAlsewi**
+
+- GitHub: [@githubhandle](https://github.com/AlselwiAisha)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+

```

Comment:
- [ ] Kindly 😃 it will be better if you increase the contacts of you here to let the users can communicate with you for example you create a LinkedIn account and Twitter and put the links for the accounts here to let the users in the future see your profile in the accounts and see your activities in them with that you will attract their attention to you 💯🚀